docs/setup-cluster/deploy-cluster/slurm/hpc-with-agent.rst

```diff
@@ -0,0 +1,153 @@
+.. _hpc-with-agent:
+
+####################
+ Agent on Slurm/PBS
+####################
+
+As an alternative to using the HPC Launcher, you may instead utilize the Determined agent. In this
+usage model, the system administrator creates a custom resource pool for each Determined user. You
+then start a Determined agent on one or more compute nodes of the cluster using Slurm or PBS
+commands to provide the resources for your resource pool. As work is submitted to this resource
+pool, it is distributed to the set of available agents. If your Slurm/PBS job is terminated (for
+example due to a time limit) before your Determined work is completed, your Determined work remains
+in your resource pool until additional agents are started. You may add additional resources to your
+resource pool by starting additional agents on your cluster. If your Determined work is complete
```

Comment:
The starting of additional agents requires that they all use the same config file as the previously started agents, so that they all reference the same resource pool (which is specified in the config file), correct?

Reply:
Right.   However many you start will all just contribute agents/slots to the resource pool.

Reply:
I don't think there is any security on offering resources, however, the agent will not process jobs for a determined account unless the uid/gid configured for that account match that of the running user.   That is why RBAC is suggested to limit the resource pool visibility by those that shouldn't be using them.